src/helpers/telegram/commands/restore.js

```diff
@@ -57,7 +57,7 @@ module.exports = (ctx) => {
 							controller.query.deleteQuery('quest', 'id', target.id).catch((O_o) => {}),
 							controller.query.deleteQuery('incident', 'id', target.id).catch((O_o) => {}),
 						]).then((x) => {
-							const query = fs.readFileSync(path.join(__dirname, '../../commando/commands', `/filterBackups/${args[0]}.sql`), 'utf8').replace('{{ target }}', target.id)
+							const query = fs.readFileSync(path.join(__dirname, '../../commando/commands', `/filterBackups/${args[0]}.sql`), 'utf8').replace(/'{{ target }}'/g, target.id)
```

Comment:
I think quotes would break this 

`/{{ target }}/g`  pls

Reply:
also pr's base to develop pls

Reply:
I only tested with quotes.

Reply:
it works there because you're replacing the string in the query with a number. 

for other Id's like webhooks where the id is a varchar it would error as those aren't numbers, but strings